lib/player.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'ui.rb'
+require_relative 'board.rb'
+
+class Player
+  attr_accessor :sign
+  def initialize
+    self.sign = nil
```

Comment:
For instance variables we should instead use `@` instead of `self`. So for here we better call it `@sign`.